autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -268,10 +348,15 @@ impl BigTableClientImpl {
         req: bigtable::MutateRowRequest,
     ) -> Result<(), error::BigTableError> {
         let bigtable = self.pool.get().await?;
-        bigtable
-            .conn
-            .mutate_row_async_opt(&req, call_opts(self.metadata.clone()))
-            .map_err(error::BigTableError::Write)?
+        retry_policy(self.settings.retry_count)
```

Comment:
Should make this a method

```suggestion
        self.retry_policy()
```

Reply:
(Remembers why it's not): 
I use the same `retry_policy` for the `heath_check` which doesn't take a `BigClientTableImpl`. 
I may need to restructure things a bit to do that.

Reply:
I noticed that, I'm fine with a one off for it. We have an open issue to rework the health check so that whole method will probably change quite a bit sooner rather than later anyway.

Reply:
Nope, gonna push back on this one. 
The function takes one parameter, and we're using it in different paths, so making this dependent on `BigTableClientImpl` would over complicate things right now. 

We might want to revisit the structure of all of this later, but that should be a different ticket.

Reply:
Yeah, there's a lot of "Get It Working" in this. There's lots of room for fit and finish later.